specification/monitor/resource-manager/readme.python.md

```diff
@@ -24,6 +24,18 @@ no-namespace-folders: true
 python-base-folder: monitor/azure-mgmt-monitor/azure/mgmt/monitor
 modelerfour:
   lenient-model-deduplication: true
+batch:
+  - AMCS: true
+```
+
+These settings apply when `--track2` and `--AMCS` is specified on the command line.
```

Comment:
why do you need this --AMCS ? Also could you follow the sample files template https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/documentation/samplefiles/readme.python.md

Reply:
I need to pick specific files to build amcs vender sdk.  I followed the doc in autorest.az [how-to-author-readme-files-for-multi-extension](https://github.com/Azure/autorest.az/blob/master/doc/how-to-author-readme-files-for-multi-extension.md).

Reply:
Then you shouldn't add the batch part into track2 section, it means you want generate  the AMCS package as long as --track2 is passing